File: test/data/cs/test06/ConsoleOutControllee.cs
public class ConsoleOutControllee : IControllee
{
    public void DoSaveResult()
    {
        Console.WriteLine("SaveResult");
    }
    public void NoTransition(string state, string transition)
    {
        Console.WriteLine($"NoTransition({state}: {transition})");
    }
    public void OverTransition(string transition)
    {
        Console.WriteLine($"OverTransition({transition})");
    }
}

File: test/data/typescript/test06/ConsoleOutControllee.ts
import {IControllee} from "./IControllee"
export class ConsoleOutControllee implements IControllee
{
    public DoSaveResult(): void
    {
        console.log("SaveResult");
    }
    public NoTransition(state: string, transition: string): void
    {
        console.log(`NoTransition(${state}: ${transition})`);
    }
    public OverTransition(transition: string): void
    {
        console.log(`OverTransition(${transition})`);
    }
}